libdnf/rpm/solv/map_iterator.hpp

```diff
@@ -152,6 +155,28 @@ inline SolvMapIterator SolvMapIterator::operator++(int) noexcept {
     return it;
 }
 
+inline void SolvMapIterator::jump(PackageId id) noexcept {
+    if (id.id < 0) {
+        begin();
+        return;
+    }
+
+    const unsigned char * current = map->map + (id.id >> 3);
+
+    if (current >= map_end) {
+        end();
+        return;
+    }
+
+    current_value = id;
+    map_current = current;
+
+    // If the element with requested id does not exist in the map, it moves to the next.
+    if (!(*current & (1 << (id.id & 7)))) {
```

Comment:
This line with line 164 makes the same job like `MAPTST(m, n) ((m)->map[(n) >> 3] & (1 << ((n) & 7))) ` in libsolv. The question is - shall we mention it to somehow link it for future?

Reply:
The pointer to `(m)->map[(n) >> 3]` is precomputed to `current`. And the `current` value is needed for test before. However, the `jump()` method is called usually once at the beginning of the cycle. 
So using `MAPTST()` or `map_tst()` has a negligible effect. And it can be optimized by compiler.

Should I use `MAPTST()` for better readibility at line 175? Oh, I understand your comment now. You want only to add a comment line with the information.

Reply:
No, do not use MAPTST(), I just want to create a not that it is similar like libsolv macro. In case that libsolv will have some new magic wi can track it and improve our code. It is a minor comment that you can ignore :).